static/js/test_util.ts

```diff
@@ -9,3 +12,43 @@ export const defaultFormikChildProps: FormikState<any> = {
   status:       null,
   submitCount:  0
 }
+
+export const TEST_MARKDOWN = `## A heading
```

Comment:
What do you think about implementing this as a separate `.md` file? `static/js/resources/example-markdown.md` or something like that. I'm thinking we would keep this constant but set the value by importing those file contents. You'd know better than me regarding what that would take in webpack, and there might be some drawbacks I'm not seeing

Reply:
this file isn't included in webpack so we should be good to do that, I'll give it a try.

Reply:
ahhh this is more difficult than it should be, so if it's alright with you I think it's best to keep this all-typescript